symengine/parser.h

```diff
@@ -7,6 +7,7 @@ namespace SymEngine
 {
 
 RCP<const Basic> parse(const std::string &s, bool convert_xor = true);
+RCP<const Basic> parse_julia(const std::string &s, bool convert_xor = true);
```

Comment:
I don't think we need a special function for julia, but we should add a function like `basic_parse2` to expose sending `local_parser_constants` to `parse` C++ function.

Reply:
I don't see a string->basic map type in the cwrapper, should I add a `CMapStrBasic` for this?

Reply:
Yes. See also https://github.com/symengine/symengine/issues/1378